### ISSUE TITLE
How to enable enhanced settings?

### ISSUE THREAD

**Issue by yslupdates** [c0re100/qBittorrent-Enhanced-Edition#303]
### qBittorrent version and Operating System
4.3.9 + latest raspbian on pi 4

How do I access the enhanced features like Auto Update Public Trackers List? I'm currently having trouble finding it within the settings section.

**Comment by abcfy2** [comment]
![图片](https://user-images.githubusercontent.com/5327677/143726389-c8489aeb-7f2f-4ed0-9703-280404b16c3c.png)
![图片](https://user-images.githubusercontent.com/5327677/143726401-91845cb4-9097-4d75-a90a-6ff0b4ce6e50.png)

**Comment by yslupdates** [comment]
Hi, apologies. I am using it via webUI from a debian raspberry pi 4. Should be latest everything as of today. Please see my screenshots below of webUI

![Screen Shot 2021-11-27 at 9 19 32 PM](https://user-images.githubusercontent.com/53108354/143730665-3a06e9da-a439-4fed-9c7f-bca5cd67c2ce.png)
![Screen Shot 2021-11-27 at 9 20 31 PM](https://user-images.githubusercontent.com/53108354/143730691-cb8a205c-3ca1-4bbc-a168-1f8ef4d5d075.png)

**Comment by abcfy2** [comment]
Seems you are using official qbittorrent, would you please double check it?

Did you download and run qbittorrent from https://github.com/c0re100/qBittorrent-Enhanced-Edition/releases/tag/release-4.3.9.10 ?

What's the version?

```sh
/path/to/qbittorrent-nox --version
```

Should be:

```sh
qBittorrent v4.3.9.10
```

**Comment by yslupdates** [comment]
I followed the guide to add stable repo for raspbian 11 here:

https://github.com/c0re100/qBittorrent-Enhanced-Edition#repository

What command should I do for this? Not sure how to manually install, used to having repo handle it.

**Comment by abcfy2** [comment]
You may check version from webui: Help --> About:

![图片](https://user-images.githubusercontent.com/5327677/143731196-7caa3b34-2fda-4337-aa7b-11d8d2aa4b14.png)

**Comment by yslupdates** [comment]
Yes, confirmed 4.3.9 and not 4.3.9.10. Please tell me how can I install the .10 version either manually with deb or through apt and repo.

**Comment by abcfy2** [comment]
If you are using Ubuntu, you may try to add PPA: https://launchpad.net/~poplite/+archive/ubuntu/qbittorrent-enhanced

Otherwise you could download static linked build from: https://github.com/c0re100/qBittorrent-Enhanced-Edition/releases/tag/release-4.3.9.10 and unzip it and run it manually.

**Comment by yslupdates** [comment]
I believe I need this file below, what are the commands to install this through ssh? I'm on a rapsberry Pi 4


https://github.com/c0re100/qBittorrent-Enhanced-Edition/releases/download/release-4.3.9.10/qbittorrent-nox_aarch64-linux-musl_static.zip

**Comment by abcfy2** [comment]
You should download it in your machine, of couse you can do it through ssh.

E.g:

```sh
wget https://github.com/c0re100/qBittorrent-Enhanced-Edition/releases/download/release-4.3.9.10/qbittorrent-nox_aarch64-linux-musl_static.zip

unzip qbittorrent-nox_aarch64-linux-musl_static.zip

./qbittorrent-nox
```

Also if you want to run it as service, you may use systemd: https://www.freedesktop.org/wiki/Software/systemd/

**Comment by yslupdates** [comment]
Still not working out. I recently switched a new Pi 4 4GB and ran these commands which were linked from the README:

`echo 'deb http://download.opensuse.org/repositories/home:/nikoneko:/test/Raspbian_11/ /' | sudo tee /etc/apt/sources.list.d/home:nikoneko:test.list
curl -fsSL https://download.opensuse.org/repositories/home:nikoneko:test/Raspbian_11/Release.key | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/home_nikoneko_test.gpg > /dev/null
sudo apt update
sudo apt install qbittorrent-enhanced`

After I install I don't see any enhancements done and it's still 4.3.9 and not 4.3.9.10

Am I missing something from these instructions?

**Comment by abcfy2** [comment]
You should also install `qbittorrent-enhanced-nox`

**Comment by Kolcha** [comment]
first of all `qbittorrent-enhanced` and official `qbittorrent` packages can't co-exist. before installing one of them another must be uninstalled.
the second thing is just to clarify, packages with `-nox` suffix is daemon version of qBittorrent which has only WebUI, packages without such suffix contain qBittorrent (including Enhanced) with both desktop UI and WebUI (WebUI is disabled by default).
and the third, mentioned instructions about packages installation are not written by me, they are just generated by server I use, and they contain only one package name, regardless few packages are available. you can find all available packages for Raspbian_11 here: https://download.opensuse.org/repositories/home:/nikoneko:/test/Raspbian_11/

so, as mentioned above, `qbittorrent-enhanced-nox` package should be installed instead of `qbittorrent-enhanced` mentioned in instructions to get WebUI-only version.
